standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/jdbc/queries/LatestTxnIdInConflictHandler.java

```diff
@@ -65,9 +66,12 @@ public String getParameterizedQueryString(DatabaseProduct databaseProduct) throw
 
   @Override
   public SqlParameterSource getQueryParameters() {
+    List<String> types = new ArrayList<>(2);
+    types.add(OperationType.UPDATE.getSqlConst());
+    types.add(OperationType.DELETE.getSqlConst());
     return new MapSqlParameterSource()
         .addValue("txnId", txnId)
-        .addValue("types", List.of(OperationType.UPDATE.getSqlConst(), OperationType.DELETE.getSqlConst()), Types.CHAR)
+        .addValue("types", types, Types.CHAR)
```

Comment:
what was the issue here?

Reply:
The java.util.List does not have .of() method. I accidentally imported com.sun.tools.javac.util.List which caused runtime issues, see [HIVE-28004](https://issues.apache.org/jira/browse/HIVE-28004). Honestly I have no idea why the tests did not fail on this one.